wordle-web/plugins/vuetify.ts

```diff
@@ -1,38 +1,121 @@
 // import this after install `@mdi/font` package
 import "@mdi/font/css/materialdesignicons.css";
+import colors from "vuetify/lib/util/colors"; // Corrected import path
 
 import "vuetify/styles";
-import { createVuetify } from "vuetify";
+import { createVuetify, type ThemeDefinition } from "vuetify";
```

Comment:
Adding comment based on my main review comment: I think we just need two original color schemes, but they need to be dark/light mode toggleable.

Reply:
we had a great time adding more than 2, but thank you for the tip we have fixed the dark/light mode toggleable!